content/asset-tracking/using-the-sdks.textile

```diff
@@ -600,8 +601,82 @@ subscriber.nextLocationUpdateIntervals
 
 See also the SDK reference docs for more "Subscriber properties":https://sdk.ably.com/builds/ably/ably-asset-tracking-android/main/dokka/subscribing-sdk/subscribing-sdk/com.ably.tracking.subscriber/-subscriber/index.html.
 
+h2(#location-animator). Location animator
+
+The location animator is an extension for the subscriber that provides smooth location updates animation. 
+
+The location animator receives location updates from the publisher and creates animation steps from them. These steps are then applied to an ongoing animation. 
+
+The animation duration is then recalculated as the sum of location update interval (the time to next location update from the publisher) and the intentional delay. The intentional delay allows for network issues or delays in the location updates. For example, if the intentional delay was set to two seconds, then a two second buffer will help the animation remain smooth and continuous, even if the publisher did not send a location update within the specified time, with a two second margin.
+
+The location animator then recalculates each step duration to evenly take up the animation duration, so the overall animation of the trackable is smooth.
+
+Each step is animated at 60 FPS, and this applies to both the map marker and the camera position.
+
+More information can also be found in the location animator "reference docs":https://sdk.ably.com/builds/ably/ably-asset-tracking-android/main/dokka/ui-sdk/ui-sdk/com.ably.tracking.ui.animation/-location-animator/index.html.
+
+h3(#location-animator-create-instance). Create an instance
+
+Creating an instance of the location animator:
+
+```[kotlin]
+val locationAnimator: LocationAnimator = CoreLocationAnimator(
+  intentionalAnimationDelayInMilliseconds,
+  animationStepsBetweenCameraUpdates
+) 
+```
+
+There are two optional parameters:
+
+- intentionalAnimationDelayInMilliseconds := The higher the delay, the more buffer there is to allow for unexpected delays of location updates from the publisher. Also moves the actual position of the trackable backwards in time. The higher the setting the more network issues and delays are allowed for, but the position updates will be less realtime in nature. By default this is set to two seconds, but you should test what value works best for your use case.
+- animationStepsBetweenCameraUpdates := By default, when a step is being animation, the camera position is also updated. But this can be changed so that the camera position updates, for example, on every five location updates.  
```

Comment:
Is it a typo? 🤔 
```suggestion
- animationStepsBetweenCameraUpdates := By default, when a step is being animated, the camera position is also updated. But this can be changed so that the camera position updates, for example, on every five location updates.  
```